rmf_schedule_visualizer/src/VisualizerData.cpp

```diff
@@ -222,4 +248,46 @@ std::vector<std::vector<
   return conflicts;
 }
 
+std::vector<Element> VisualizerDataNode::get_negotiation_trajectories(
+  uint64_t conflict_version, const std::vector<uint64_t>& sequence) const
+{
+  std::vector<Element> trajectory_elements;
+
+  auto table_view = _negotiation->table_view(conflict_version, sequence);
+  if (!table_view)
+  {
+    RCLCPP_WARN(
+      this->get_logger(), "table_view for conflict %d not found!",
+      conflict_version);
+    return trajectory_elements;
+  }
+
+  rmf_traffic::RouteId route_id = 0;
+  auto add_route = [&](rmf_traffic::ConstRoutePtr route_ptr,
```

Comment:
```suggestion
  const auto add_route = [&](rmf_traffic::ConstRoutePtr route_ptr,
```

On a different note, this lambda can be const-qualified.

Reply:
add const 3b1c580